kolibri/core/assets/src/vue/error-box/index.vue

```diff
@@ -21,9 +21,9 @@
     $trNameSpace: 'error',
     $trs: {
       errorHeader: 'Error',
-      errorButtonLabel: 'Hide Error',
+      errorButtonLabel: 'Hide error',
       // eslint-disable-next-line
-      explanation: `Sorry, we've encountered an issue. You may need to refresh the page or click one of the main navigation links.`,
+      explanation: `Unknown issue. You may need to refresh`,
```

Comment:
Yep, I believe my suggestion was actually in line with the first guideline.
Good point about the administrator!